### PR TITLE
Add property timezone as a database field

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
Meters need to know the timezone in order to have the correct time stamps in the meter data. It made sense to add the timezone string to the property which makes the assumption that the meters are all within the property and that the property does not span multiple time zones.

#### What's this PR do?
* Add property_timezone
* Update `example-data-properties.xlsx` with a column for timezones
* Format of the timezone needs to follow the `pytz.all_timezones` format. 

#### How should this be manually tested?
* import new data that has a column for timezone
* map to new field
*Note that the timezone string is not validated. Downstream code will need to ensure the string is in correct format*

#### What are the relevant tickets?
#3243 

#### Screenshots (if appropriate)
 
image

### CODE REVIEW

**Comment by RDmitchell** [comment]
Sensors also need the timezone functionality.

**Comment by haneslinger** [comment]
> Sensors also need the timezone functionality.

This is for the sensors. They will all share their property's timezone.

**Comment by RDmitchell** [comment]
Perfect !

**Review by haneslinger — APPROVED** [review]
Thank you!

**Comment by RDmitchell** [comment]
Thank you !!

File: seed/models/properties.py [hunk 1]
```diff
@@ -131,7 +131,23 @@ def copy_meters(self, source_property_id, source_persists=True):
 
 
 class PropertyState(models.Model):
-    """Store a single property. This contains all the state information about the property"""
+    """Store a single property. This contains all the state information about the property
+
+    For property_timezone, use the pytz timezone strings. The US has the following and a full
+    list can be creates by running calling pytz.all_timezones in Python:
```
Comment (nllong) [inline seed/models/properties.py:137]:
```suggestion
    list can be created by calling pytz.all_timezones in Python:
```

**Pull request merged**